eng/common/scripts/Verify-Links.ps1

```diff
@@ -165,7 +174,17 @@ function CheckLink ([System.Uri]$linkUri)
       }
     }
   }
-  $checkedLinks[$linkUri] = $true;
+  
+  # Check if link uri includes locale info.
+  if ($checkLinkGuidance -and ($linkUri -match $locale)) {
+    LogWarning "DO NOT include locale $locale information in links: $linkUri."
+    if ($linkValid) {
+      $script:badLinks += $linkUri
```

Comment:
Lets please move all the additions of badLinks from this function as well and do it where we call CheckLink